internal/app/connectconformance/testsuites/data/connect_idempotency.yaml

```diff
@@ -12,7 +12,7 @@ relevantCodecs:
   - CODEC_PROTO
 testCases:
 - request:
-    testName: idempotent unary success proto
+    testName: unary/success
```

Comment:
Ditto. Only unary calls can use GET, so this feels redundant. Also, this isn't really testing idempotency so much as it is testing the GET HTTP method. So maybe rename the test suite and file to "Connect with GET"